BEIMA.Backend/DeviceFunctions/DeleteDevice.cs

```diff
@@ -36,11 +41,35 @@ public static class DeleteDevice
             var mongo = MongoDefinition.MongoInstance;
             var deviceId = new ObjectId(id);
```

Comment:
Looks like I must've missed this, but we should run an ObjectId.TryParse on the id before assigning it here, and return the invalid id message if it's not good.

Reply:
It does get try parsed right above line 36. I can change the assignment to ObjetId.Parse(id) though if that's better

Reply:
Oh, woops, didn't see that, should be fine then.